raycicmd/make.go

```diff
@@ -96,7 +96,7 @@ func makePipeline(repoDir string, config *config, info *buildInfo) (
 				return nil, fmt.Errorf("parse pipeline file %s: %w", file, err)
 			}
 
-			if !tagFilters.hit(g.Tags) {
+			if len(g.Tags) > 0 && !tagFilters.hit(g.Tags) {
```

Comment:
len check is already done inside hit?

Reply:
yes. I added this one first, but then figured that it is clearer to add inside.

removed this one now.